gsa/src/web/pages/extras/feedstatuspage.js

```diff
@@ -162,6 +171,34 @@ const FeedStatus = () => {
                       </Link>
                     </IconDivider>
                   )}
+                  {feed.feed_type === GVMD_DATA && (
+                    <IconDivider>
+                      <Link to="policies">
+                        <IconDivider align={['start', 'center']}>
+                          <PolicyIcon size="medium" />
+                          <span>Compliance Policies</span>
```

Comment:
These titles should be translatable. While you're at it, please also make the old titles translatable if applicable.